cirq-google/cirq_google/engine/engine_client_test.py

```diff
@@ -467,6 +469,133 @@ def test_create_job(client_constructor):
         )
 
 
+@mock.patch.dict(os.environ, clear='CIRQ_TESTING')
+@mock.patch.object(quantum, 'QuantumEngineServiceAsyncClient', autospec=True)
+def test_create_job_with_run_name_and_device_config_name_and_no_processor_id_throws(
```

Comment:
WDYT of grouping all your validation tests together in a parametrized test, with different `create_job` args and error messages?